src/vaadin-split-layout.html

```diff
@@ -332,6 +343,14 @@
           }
         }
 
+        /**
+         * Can be called to manually notify a resizable and its descendant
+         * resizables of a resize change.
+         */
+        notifyResize() {
+          super.notifyResize();
+        }
```

Comment:
Should we add some extra comment here saying why we have this override (for generated type defs) so it doesn't get accidentally removed later as it looks a bit unnecessary?

Reply:
I think that's a good idea. Otherwise there's a risk that later after we've forgotten this, we think this serves no purpose and remove it.

Reply:
Done.